index.html

```diff
@@ -40,20 +40,57 @@ <h2>Experience</h2>
             <h2>Connect</h2>
             <ul style="list-style-type:none;">
                 <li><a
-                     href="https://github.com/Startnow08/Bianca-Goodwin-Jupiter">Github account 
+                     href="https://github.com/Startnow08/Bianca-Goodwin-io-CodetheDream">Github account 
                 </a>
                 </li>
             </ul>
         </section>
 
         <section id="skills">
             <h2>Skills</h2>
-            <ul id="skill-ul"></ul>
+            <ul></ul>
         </section>
 
         <section id="projects">
             <h2>Projects</h2>
         </section>
+            <h2>Leave a Message</h2>
+            <form id="MessageForm" name="leave_message" autocomplete="on">
+                <label for="usersNameId">Users Name:</label>
+                <input
+                type="text"
+                name="usersName"
+                id="usersNameId"
+                required="true"
+                /> <br />
+                <label for="emailID">Email: </label>
+                <input
+                type="email"
+                name="UsersEmail"
+                id="emailID"
+                required="true"
+                autocomplete="on"
+                /> <br />
+                <label for="usersMessageID">Users Message: </label>
+                <input
+                    type="text"
+                    name="usersMessage"
+                    requried="true"
```

Comment:
You've got a typo here

```suggestion
                    required="true"
```